docs/tutorial/intro_to_resqml.rst

```diff
@@ -0,0 +1,83 @@
+
+The current state of play of the main providers of modelling software to BP is:
+
+* Emerson (RMS): From version 12, RMS has some RESQML import and export capability. Not all RESQML classes are yet supported. However, plug-ins are being released on a regular basis with new RESQML functionality added each time. There is a close working relationship between bp and Emerson.
+* Schlumberger (Petrel): There is an add on module available which allows Petrel to import and export in RESQML format. A successful export has been made within BP, though not all object types are included in the RESQML model.
+* Halliburton / Landmark (Nexus): Nexus cannot currently read or write RESQML models. However, development is underway to allow grids, grid properties and some other inputs to be sourced directly from a RESQML dataset, using a syntax similar to that for reading from a vdb. There is a good working relationship between bp and the Halliburton developers.
+* DGI (Coviz): The Coviz API includes a RESQML import function that will convert from RESQML to DGI's proprietary c3grd format. It also includes an export function for converting in the opposite direction. It works well for IJK Grid geometries, though there appear to be some bugs needing to be fixed. It can handle coordinate reference systems. Although it can also process grid properties, it does so in a rudimentary manner that will need to be enhanced. (It currently does not use the standard RESQML property kinds or units of measure.) BP has a close working relationship with DGI and they are the most responsive of our software providers.
+* F2I (FESAPI): You may not have heard of this small software consultancy. However, the latter 3 companies listed above are using the FESAPI interface to read and write RESQML models. This is the only API (application programming interface) available as an alternative to in-house development. It consists mainly of C++ code generated from the RESQML xsd schema files, providing conversion between C++ objects and RESQML xml trees. It primarily targets a Windows runtime environment. SWIG wrappers are available for onward conversion to Java (and could potentially be used to support Python). However, standard Python libraries provide much of the functionality to convert between xml and Python objects directly, as well as good hdf5 to numpy interaction, so FESAPI is not an ideal solution if working in Python.
```

Comment:
I think we should remove this section, as it doesn't seem appropriate for a general-purpose package docs (and perhaps do a `git rebase` to avoid it ending up in the git history)